daprdocs/content/en/developing-applications/develop-components/pluggable-components/develop-pluggable.md

```diff
@@ -105,6 +105,167 @@ After generating the above state store example's service scaffolding code using
+   new Google.Rpc.BadRequest.Types.FieldViolation
+       {        
+         Field = "etag",
+         Description = des
+       });
+
+var baseStatusCode = Grpc.Core.StatusCode.FailedPrecondition;
+var status = new Google.Rpc.Status{    
+   Code = (int)baseStatusCode
+};
+
+status.Details.Add(Google.Protobuf.WellKnownTypes.Any.Pack(badRequest));
+
+var metadata = new Metadata();
+metadata.Add("grpc-status-details-bin", status.ToByteArray());
+throw new RpcException(new Grpc.Core.Status(baseStatusCode, "fake-err-msg"), metadata);
+```
+
+**Etag Invalid**
+
+```csharp
+var badRequest = new BadRequest();
+var des = "The ETag field must only contain alphanumeric characters";
+badRequest.FieldViolations.Add(
+   new Google.Rpc.BadRequest.Types.FieldViolation
+   {
+      Field = "etag",
+      Description = des
+   });
+
+var baseStatusCode = Grpc.Core.StatusCode.InvalidArgument;
+var status = new Google.Rpc.Status
+{
+   Code = (int)baseStatusCode
+};
+
+status.Details.Add(Google.Protobuf.WellKnownTypes.Any.Pack(badRequest));
+
+var metadata = new Metadata();
+metadata.Add("grpc-status-details-bin", status.ToByteArray());
+throw new RpcException(new Grpc.Core.Status(baseStatusCode, "fake-err-msg"), metadata);
+```
+
+**Bulk Delete Row Mismatch**
+
+```csharp
+var errorInfo = new Google.Rpc.ErrorInfo();
+
+errorInfo.Metadata.Add("expected", "100");
+errorInfo.Metadata.Add("affected", "99");
+
+var baseStatusCode = Grpc.Core.StatusCode.Internal;
+var status = new Google.Rpc.Status{
+    Code = (int)baseStatusCode
+};
+
+status.Details.Add(Google.Protobuf.WellKnownTypes.Any.Pack(errorInfo));
+
+var metadata = new Metadata();
+metadata.Add("grpc-status-details-bin", status.ToByteArray());
+throw new RpcException(new Grpc.Core.Status(baseStatusCode, "fake-err-msg"), metadata);
+```
+
+{{% /codetab %}}
+
+ <!-- Java -->
+{{% codetab %}}
+
+**ETag Mismatch**
+
+```java
+
+```
+
+**ETag Invalid**
+
+```java
+
+```
+
+**Bulk Delete Row Mismatch**
+
+```java
+
+```
+
+{{% /codetab %}}
+
+ <!-- Go -->
+{{% codetab %}}
+
+**ETag Mismatch**
+
+```go
+st := status.New(codes.FailedPrecondition, "fake-err-msg")
+desc := "The ETag field must only contain alphanumeric characters"
+v := &errdetails.BadRequest_FieldViolation{
+	Field:       etagField,
+	Description: desc,
+}
+br := &errdetails.BadRequest{}
+br.FieldViolations = append(br.FieldViolations, v)
+st, err := st.WithDetails(br)
+```
+
+**ETag Invalid**
+
+```go
+st := status.New(codes.InvalidArgument, "fake-err-msg")
+desc := "The ETag field must only contain alphanumeric characters"
+v := &errdetails.BadRequest_FieldViolation{
+	Field:       etagField,
+	Description: desc,
+}
+br := &errdetails.BadRequest{}
+br.FieldViolations = append(br.FieldViolations, v)
+st, err := st.WithDetails(br)
+```
+
+**Bulk Delete Row Mismatch**
+
+```go
+st := status.New(codes.Internal, "fake-err-msg")
+br := &errdetails.ErrorInfo{}
+br.Metadata = map[string]string{
+	affected: "100",
+	expected: "99",
+}
```

Comment:
@mcandeia Isn't this backwards? Like, we **expected** this delete to impact **100** but but 1 failed and thus only expected were expected?

If so, other samples will need to be updated too

Reply:
good catch, yes